AlphaWallet/AddMultipleCustomRpc/AddMultipleCustomRpcViewController.swift

```diff
@@ -192,10 +192,10 @@ extension AddCustomChain {
 
     convenience init(_ customRpc: CustomRPC, restartHandler: RestartQueueHandler, networkService: NetworkService, analytics: AnalyticsLogger) {
         let defaultDecimals = 18
-        let explorerEndpoints: [String]?
+        let explorerEndpoints: [WalletAddEthereumChainObject.ExplorerUrl]?
```

Comment:
@oa-s is this a separate change?

Reply:
Nope, tester with dapp from example app, but we could extract it